cms/static/js/certificates/views/signatory_editor.js

```diff
@@ -0,0 +1,144 @@
+            return this.model.collection.indexOf(givenModel);
+        },
+
+        loadTemplate: function(name) {
+            // Retrieve the corresponding template for this model
+            return TemplateUtils.loadTemplate(name);
+        },
+
+        getTotalSignatoriesOnServer: function() {
+            // Retrieve the count of signatories stored server-side
+            var count = 0;
+            this.model.collection.each(function( modelSignatory) {
+                if(!modelSignatory.isNew()) {
+                    count ++;
+                }
+            });
+            return count;
+        },
+
+        render: function() {
+            // Assemble the editor view for this model
+            var attributes = $.extend({
+                modelIsValid: this.model.isValid(),
+                error: this.model.validationError
+            }, this.model.attributes, {
+                signatory_number: this.getModelIndex(this.model) + 1,
+                signatories_count: this.model.collection.length,
+                isNew: this.model.isNew(),
+                is_editing_all_collections: this.isEditingAllCollections,
+                total_saved_signatories: this.getTotalSignatoriesOnServer()
+            });
+            return $(this.el).html(this.template(attributes));
+        },
+
+        setSignatoryName: function(event) {
+            // Update the model with the provided data
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set(
+                'name',
+                this.$('.signatory-name-input').val()
+            );
+            this.eventAgg.trigger("onSignatoryUpdated", this.model);
+        },
+
+        setSignatoryTitle: function(event) {
+            // Update the model with the provided data
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set(
+                'title',
+                this.$('.signatory-title-input').val()
+            );
+            this.eventAgg.trigger("onSignatoryUpdated", this.model);
+        },
+
+        setSignatoryOrganization: function(event) {
+            // Update the model with the provided data
+            if (event && event.preventDefault) { event.preventDefault(); }
+            this.model.set(
+                'organization',
+                this.$('.signatory-organization-input').val()
+            );
+            this.eventAgg.trigger("onSignatoryUpdated", this.model);
+        },
+
+        deleteItem: function(event) {
+            // Remove the specified model from the collection
+            if (event && event.preventDefault) { event.preventDefault(); }
+            var certificate = this.model.get('certificate');
+            var model = this.model;
+            var self = this;
+            var confirm = new PromptView.Warning({
+                title: gettext('Are you sure you want to delete "'+model.get('title') +'" as a signatory?'),
```

Comment:
You can't do concatenate strings like this for translation. You need to use a placeholder and the interpolate function.

http://edx-developer-guide.readthedocs.org/en/latest/internationalization/i18n.html